TrigScint/src/TrigScint/QIEDecoder.cxx

```diff
@@ -63,15 +63,6 @@ void QIEDecoder::produce(framework::Event &event) {
   ldmx_log(debug) << "QIEDecoder: produce() starts! Event number: "
                   << event.getEventHeader().getEventNumber();
 
-  //	ldmx::EventHeader *eh = (ldmx::EventHeader*)event.getEventHeaderPtr();
-  // eh->setIsRealData(isRealData_); //doesn't help
-
-  /*   // comment for now, requires pushing change ot EventHeader
-  event.getEventHeader().setIsRealData(isRealData_);
```

Comment:
I'd rather leave this in, still commented; we want to be able to set this somehow. Or we remove it but then make an issue for having a way to indicate that something is real data. (But you see the logic, if it weren't still there, I'd have forgotten about it...)

Reply:
Here you go: https://github.com/LDMX-Software/ldmx-sw/issues/1428
The implementation doesnt seem to complicated, we just need to add another function to the event header right? If yes, I can try to do this next.